httpklient/src/main/kotlin/no/nav/aap/komponenter/httpklient/httpclient/RestClient.kt

```diff
@@ -79,6 +89,49 @@ public class RestClient<K>(
         return executeRequestAndHandleResponse(httpRequest, mapper)
     }
 
+    public fun <R> retryableGet(uri: URI, request: GetRequest, mapper: (K, HttpHeaders) -> R, maxRetries: Int = 2): R? {
+        val httpRequest = buildRequest(uri, request)
+
+        return retry(maxRetries) { executeRequestAndHandleResponse(httpRequest, mapper) }
+    }
+
+    public fun <T : Any, R> retryablePost(uri: URI, request: PostRequest<T>, mapper: (K, HttpHeaders) -> R, maxRetries: Int = 2): R? {
+        val httpRequest = buildRequest(uri, request)
+
+        return retry(maxRetries) { executeRequestAndHandleResponse(httpRequest, mapper) }
+    }
+
+    private fun <R> retry(retries: Int, function: () -> R?): R? {
+        try {
+            return function()
+        } catch (ex: Exception) {
+            if (retryExceptions.contains(ex::class) && retries - 1 > 0) {
+                log.info("Feilet kall, retryer : {}", ex.message)
+                return retry(retries - 1, function)
```

Comment:
Kanskje legge inn en pitteliten wait her (som ideelt øker for hver retry)?

Om det er nettverksproblemer, vil jo ikke retry hjelpe om man prøver igjen etter 0 sek)

Reply:
Tja, det vil jo være en timeout på noen sekunder fra den forsøkte til den vil feile - så tror nok det bør fungere greit å retrye med en gang.